website/catalog/dart/index.md

```
@@ -0,0 +1,44 @@
+id: find-direct-debug-print
+language: dart
+rule:
+  pattern:
+    context: 'void _() { debugPrint($$$ARGS); }'
+    selector: call_expression
+```
+
+This example intentionally matches only direct `debugPrint(...)` calls. For a
+copy-ready rule that also covers member, null-aware, generic, parenthesized,
+null-asserted, common single-level arrow-body, and direct cascade calls, see
+[Find Dart Calls by Callee Name](#find-dart-calls-by-callee-name). That catalog
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)